src/app/recents/recents.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, NgModule, OnInit } from '@angular/core';
+
+import { RecentsService } from '../services/recents.service';
+import { recents } from './test.data';
+import { resource } from 'selenium-webdriver/http';
+import { PropertyDisplayPipe, PropertyValueDisplayPipe } from '../pipes/propertydisplay';
+import { TabsModule } from 'ngx-bootstrap/tabs';
+
+
+@Component({
+  selector: 'app-recents',
+  templateUrl: './recents.component.html',
+  styleUrls: ['./recents.component.css']
+})
+@NgModule({
+  imports: [TabsModule.forRoot()]
+})
+export class RecentsComponent implements OnInit {
+
+  recents: Array<any>; // filtered by tab
+  recentsAll: Array<any>; // everything
+  meta: Array<any>; // metadata
+  types: Array<string>; // types of artifacts
+  loading: boolean;
+  title: String;
+
+  constructor(private recentsService: RecentsService) {
+    this.loading = true;
+    this.title = 'Recent Artifacts';
```

Comment:
This doesn't need a variable since it's a static value that doesn't change. Just use a normal `h2`.